Navigation/Screens/EditExpenses.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Text ,TextInput} from "react-native";
+import { Button } from "react-native";
+import { View } from "react-native";
+import SelectDropdown from "react-native-select-dropdown";
+import DatePicker from "react-native-date-picker";
+import { useState } from "react";
+import { StyleSheet } from "react-native";
+
+export default function EditExpenses(){
+   
+    const categories =["Food","Transportation","Utilities"]
+    const [date,setDate]=useState(new Date())
+    const [open,setOpen]= useState(false)
+    const [expense,setExpense] = useState({title:'',recipient:'',amount:'',category:'',date:''})
+    return(
+        <View>
+            <Text>fill the default pre filled values</Text>
+            <Text style={styles.text}>Title</Text>
+            <TextInput placeholder="Enter Title" style={styles.input} value={expense.title} onChangeText={(text)=>setExpense({...expense,title:text})}/>
+            <Text style={styles.text}>Recipient</Text>
+            <TextInput placeholder="Enter Recipient" style={styles.input} value={expense.recipient} onChangeText={(text)=>setExpense({...expense,recipient:text})}/>
+            <Text style={styles.text}>Amount</Text>
+            <TextInput placeholder="Enter Amount" keyboardType="numeric" style={styles.input} value={expense.amount} onChangeText={(text)=>setExpense({...expense,amount:text})}/>
```

Comment:
you can create separate functions instead of adding it inline.
like 
```
const updateAmount = (amountText: string) => {
    setExpense({...expense,amount:text})
}
```